docs/contributors.md

```diff
@@ -0,0 +1,8 @@
+# Contributors
+
+- Kat
+- Ben
+- Ankita
+- pdehaan
```

Comment:
```suggestion
The ecosystem-test-scripts repository is owned by the Ecosystem Test Engineering Team.
```

Reply:
Or if you feel extra fancy you can add a dynamic contributors list via an image with avatars:

```suggestion
The ecosystem-test-scripts repository is owned by the Ecosystem Test Engineering Team.

<a href="https://github.com/mozilla/ecosystem-test-scripts/graphs/contributors"><img src="https://contrib.rocks/image?repo=mozilla/ecosystem-test-scripts" alt="Contributors graph. Avatars provided by https://contrib.rocks"></a>
```

(that is currently empty, but will expand over time — see e.g. https://contrib.rocks/preview?repo=mozilla/normandy …)